### PR TITLE
Improved time sync protocol

### PR DESCRIPTION
**Pull request by deboer-tim**

### CODE REVIEW

File: PresCore/src/org/icpc/tools/presentation/core/internal/PresentationWindowImpl.java [hunk 1]
```diff
@@ -524,9 +522,9 @@ public void setDisplayRect(Rectangle r) {
 	 * @param time
 	 */
 	@Override
-	public void setZeroTimeMs(long time) {
+	public void setNanoTimeDelta(long time) {
```
Comment (zalenski) [inline PresCore/src/org/icpc/tools/presentation/core/internal/PresentationWindowImpl.java:525]: `long delta`?

**Review by zalenski — APPROVED** [review]

File: PresCore/src/org/icpc/tools/presentation/core/internal/PresentationWindowImpl.java (hunk 1 above)
Comment (deboer-tim) [reply PresCore/src/org/icpc/tools/presentation/core/internal/PresentationWindowImpl.java:525]: Fixed in second commit.

File: PresCore/src/org/icpc/tools/presentation/core/IPresentationHandler.java [hunk 2]
```diff
@@ -38,7 +38,7 @@
 
 	public int getFullScreenWindow();
 
-	public void setZeroTimeMs(long time);
+	public void setNanoTimeDelta(long time);
```
[the remark this answers is not in the document]
Comment (deboer-tim) [reply PresCore/src/org/icpc/tools/presentation/core/IPresentationHandler.java:41]: Fixed in second commit.

File: PresCore/src/org/icpc/tools/presentation/core/PresentationWindow.java [hunk 3]
```diff
@@ -150,7 +150,7 @@ public static PresentationWindow create() {
 	public abstract void setPresentations(long time, Presentation[] newPresentations, Transition[] newTransitions);
 
 	@Override
-	public abstract void setZeroTimeMs(long time);
+	public abstract void setNanoTimeDelta(long time);
```
[the remark this answers is not in the document]
Comment (deboer-tim) [reply PresCore/src/org/icpc/tools/presentation/core/PresentationWindow.java:153]: Fixed in second commit.

**Pull request merged**